package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "moj-product-dashboard",
+  "version": "1.0.0",
+  "description": "===================== MOJ Product Dashboard =====================",
+  "main": "index.js",
+  "dependencies": {
+    "plotly.js": "^1.11.0",
+    "whatwg-fetch": "^1.0.0"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.9.0",
+    "babel-loader": "^6.2.4",
+    "babel-polyfill": "^6.9.0",
+    "babel-preset-es2015": "^6.9.0",
+    "core-js": "^2.4.0",
```

Comment:
probably don't need `core-js` explicitly? it is probably just a dependency for one of the libs. can you give it another whirl without it?